ts_src/psetv2/updater.ts

```diff
@@ -62,6 +62,7 @@ export interface ReissuanceOpts {
   tokenAmount: number;
   tokenAddress: OutputDestination;
   tokenAssetBlinder: string | Buffer;
+  initialIssuanceBlinded?: boolean;
```

Comment:
I don't understand why we need this extra field.
IMO, it would just be enough to pass an unconf address to `assetAddress` to achieve the goal of letting the user blind only the token output, which instead MUST always be confidential to make it possible future reissaunces.

Reply:
Ok after offline discussion i understand that this flag is needed to recompute the correct token asset hash when we add the token out to the partial.

We still need to add another optional field `blindedIssuance`, used like for the issuance, to let the blinder know if the current input issuance has to be blinded or not.

I think we can remove the input methods `getIssuanceInflationKeysHash` because it relies on `blindedIssuance` flag which is not the correct one to base the computation of the token asset hash, at least for the reissuance.